clients/go/datacatalog/config.go

```diff
@@ -0,0 +1,33 @@
+package datacatalog
+
+import (
+	"context"
+
+	"github.com/flyteorg/flytestdlib/logger"
+)
+
+//go:generate pflags Config --default-var defaultConfig
+
+const ConfigSectionKey = "data-catalog"
+
+var (
+	defaultConfig = &Config{}
+
+	//TODO: Add this section when everybody has migrated
```

Comment:
Are you intending to deprecate this? https://github.com/flyteorg/flytepropeller/blob/eaf084934de5d630cd4c11aae15ecae780cc787e/pkg/controller/nodes/task/catalog/config.go#L15

And if so, should it use the same configSectionKey?